.travis.yml

```diff
@@ -1,21 +1,30 @@
 language: scala
 scala:
-  - 2.13.1
+  - 2.13.5
 jdk:
   - openjdk8
+  - openjdk11
+
+before_script:
+  # Download sbt because Travis can't find it automatically :(
+  - mkdir -p $HOME/.sbt/launchers/1.5.0/
+  - curl -L -o $HOME/.sbt/launchers/1.5.0/sbt-launch.jar https://repo1.maven.org/maven2/org/scala-sbt/sbt-launch/1.5.0/sbt-launch-1.5.0.jar
 
 script:
-  - sbt clean coverage test coverageAggregate coverageOff
-after_success:
-  # Upload coverage reports to codecov.io
-  - bash <(curl -s https://codecov.io/bash) -t 24961df6-494e-4024-b035-ad2aca2ed706
+  - sbt clean coverage +test coverageAggregate
+  # Upload coverage reports to codecov.io before checking for binary compatibility
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
This is truly bad thing to do, and I know that I've been shouted down on this, but the recent breach should be ample evidence and impetus that this isn't the way to do this.

Cache a copy of the script in and use that.  Additionally have the build try to report:
- whether it was able to successfully download a version of the script
- whether the downloaded version is fresher.

Reply:
I want to address this in a separate PR. It will affect multiple repos at once, so it might make sense to address them all at the same time.